lib/server.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Are you using this or the one in grunt-vtex?

Reply:
yup